packages/notification/test/notification.test.js

```diff
@@ -38,12 +38,21 @@ describe('vaadin-notification', () => {
     expect(getComputedStyle(notification).display).to.equal('none');
   });
 
-  it('should close on detach', () => {
+  it('should close on detach', async () => {
     expect(notification.opened).to.be.true;
     notification.remove();
+    await aTimeout(0);
     expect(notification.opened).to.be.false;
   });
 
+  it('should stay open after immediate reattach', async () => {
+    expect(notification.opened).to.be.true;
+    const parent = notification.parentNode;
+    sinon.spy(notification, '_openedChanged');
+    parent.appendChild(notification);
+    expect(notification._openedChanged.called).to.be.false;
```

Comment:
Why is the spy needed? Can't we just check `expect(notification.opened).to.be.true;`?

Reply:
Sure, I updated the test to just check the final state.